src/main/statistics_gathering/get_statistics.py

```diff
@@ -0,0 +1,117 @@
+import logging
+import os
+import pandas as pd
+
+from src.main.util import consts
+from src.main.plots import consts as stat_const
+from src.main.util.file_util import get_all_file_system_items, data_subdirs_condition, csv_file_condition, \
+    get_result_folder, create_file, change_extension_to
+from src.main.preprocessing.preprocessing import __separate_ati_and_other_files
+from src.main.preprocessing.code_tracker_handler import handle_ct_file
+
+
+log = logging.getLogger(consts.LOGGER_NAME)
+
+
+# We must have one value in a profile column else it is an incorrect case
+def __get_profile_info(ct_df: pd.DataFrame, column: consts.CODE_TRACKER_COLUMN):
```

Comment:
No, handle_ct_file, in this case, returns -1, because it is the incorrect case for us (it is a case from STATISTICS_SHOWING_KEY - INCORRECT) and we will have  always  only 1 value in the one file